ecosystem/platform/server/nginx.conf

```diff
@@ -0,0 +1,30 @@
+server {
+    listen 3000;
+    server_name community.aptoslabs.com;
```

Comment:
how would this work in other environments, like staging and dev? Can we override this somehow? E.g. consider https://community.staging.gcp.aptosdev.com/

Reply:
Updated to handle all connections to port 3000.